src/HabiticaAPIManager.js

```diff
@@ -177,6 +177,41 @@ class HabiticaAPIManager {
 		return promise;
 	}
 
+  /**
+	 * Make a POST request to the Habitica API.
+	 * Data object returned varies based on the API url called.
+	 * @param {string} baseURL - the url of the api call.
+	 * @param {HabiticaUserID} userID - the ID of the user, needed for authentication.
+	 * @param {HabiticaAPIToken} userAPIToken - the API Token for the user, needed for authentication.
+	 * @param {object} [queryParams={}] - key-value pairs for any parameters needed by the api call.
+	 * @returns {Promise<String>} Promise containing the raw API response data as a string.
+	 */
+	postRequest(baseURL, userID, userAPIToken, queryParams={}) {
+		let url = this.getQueryStringURL(baseURL, queryParams);
```

Comment:
Thanks for the PR :)

For a POST request the parameters should go in the body of the request, not in the query string.

Reply:
its resolved now!